schema/schemaHelper.h

```diff
@@ -73,12 +73,11 @@ namespace minsky
     }
     static void setPrivates
     (minsky::GodleyIcon& g, const vector<vector<string> >& data, 
-     const vector<GodleyTable::AssetClass>& assetClass,float iconScale)
+     const vector<GodleyTable::AssetClass>& assetClass)
     {
       setPrivates(g.table, data, assetClass);
-      g.m_iconScale=iconScale;
     }
```

Comment:
We can now get rid if this helper function too.

Reply:
I just tried this and it caused the variables attached to GodleyIcons to disappear. So, I've left it in.